src/Akeneo/Tool/Bundle/MeasureBundle/Resources/translations/validators.en_US.yml

```
@@ -6,5 +6,7 @@ pim_measurements:
                 limit_reached: 'You cannot create the "%measurement_family_code%" measurement family  because you have reached the limit of %limit% measurement families'
         measurement_family:
             standard_unit_code_should_exist: 'The "%standard_unit_code%" standard unit code does not exist in the list of units for the measurement family.'
+            code:
+                must_be_unique: 'This measurement family code is already used.'
```

The measurement family already exists
```suggestion
                must_be_unique: 'This measurement family code already exists.'
```